articles/storage/common/storage-network-security.md

```diff
@@ -288,6 +284,7 @@ When the "Trusted Microsoft Services" exception is enabled, the following servic
 
 |Service|Resource Provider Name|Purpose|
 |:------|:---------------------|:------|
+|Azure Backup|Microsoft.Backup|Perform and continue with scheduled and ad-hoc IaaS VM backups and restores.  [Learn more](https://docs.microsoft.com/en-us/azure/backup/backup-introduction-to-azure-backup).|
```

Comment:
Is the exception required only for VM's using UNMANAGED disks?  If so, can we call that out in the purpose here?

Reply:
We could call it out, but wouldn't that statement be non-applicable to the article since the article is about storage account firewall rules?  Just want to clarify it's not a limitation since it's not applicable to create firewall rules for managed disks as they aren't placed in customer owned storage accounts.

Reply:
How about "Perform backups and restores of un-managed disks in IAAS virtual machines. (not required for managed disks).  [Learn More]"